tests/testthat/test-test-module.R

```diff
@@ -4,6 +4,31 @@ library(promises)
 library(future)
 plan(multisession)
 
+test_that("testModule passes dots", {
+  module <- function(input, output, session, someArg) {
+    expect_false(missing(someArg))
+    expect_equal(someArg, 123)
+  }
+  testModule(module, {}, someArg = 123)
+})
+
+test_that("testModule passes dynamic dots", {
+  module <- function(input, output, session, someArg) {
+    expect_false(missing(someArg))
+    expect_equal(someArg, 123)
+  }
+
+  # Test with !!! to splice in a whole named list constructed with base::list()
+  moreArgs <- list(someArg = 123)
+  testModule(module, {}, !!!moreArgs)
+
+  # Test by splicing in a list constructed with rlang::list2() that uses := to
+  # splice in an argument name
+  argName <- "someArg"
+  moreArgs <- rlang::list2(!!argName := 123)
```

Comment:
I think this test is effectively the same as the previous one, because when `list2()` is called here, it resolves the `!!!`.

Maybe this test should instead use `!!` in the call to `testModule`?